duckscript_sdk/src/sdk/std/string/camel_case/mod.rs

```diff
@@ -0,0 +1,44 @@
+use crate::utils::pckg;
+use duckscript::types::command::{Command, CommandResult};
+
+#[cfg(test)]
+#[path = "./mod_test.rs"]
+mod mod_test;
+
+#[derive(Clone)]
+pub(crate) struct CommandImpl {
+    package: String,
+}
+
+impl Command for CommandImpl {
+    fn name(&self) -> String {
+        pckg::concat(&self.package, "CamelCase")
+    }
+
+    fn aliases(&self) -> Vec<String> {
+        vec!["camel_case".to_string()]
```

Comment:
lets also add camelcase alias so it will be consistent for all commands you are adding.

Reply:
All case names are now standardized